go/vumitools/tests/test_billing_worker.py

```diff
@@ -739,3 +744,67 @@ def test_outbound_message_session_length_custom_field(self):
             "outbound",
             session_created=False,
             session_metadata_field='foo')
+
+    @inlineCallbacks
+    def test_outbound_message_credit_cutoff_session(self):
+        self.billing_api = BillingApiMock(credit_cutoff=True)
+        dispatcher = yield self.get_dispatcher()
+
+        yield self.make_dispatch_outbound(
+            "outbound",
+            user_account="12345",
+            tag=("pool1", "1234"),
+            helper_metadata={},
+            session_event='new'
+            )
+
+        published_msg = self.ri_helper.get_dispatched_outbound()[0]
```

Comment:
We usually write this as: ``[published_msg] = self.ri_helper.get_dispatched_outbound()`` so that we raise an error if there isn't exactly one message.